pkg/event-manager/drivers/k8s_backend.go

```diff
@@ -92,9 +92,11 @@ func (k *k8sBackend) makeDeploymentSpec(driver *entities.Driver) (*v1beta1.Deplo
 		inputMap[key] = val
 	}
 
-	// secrets are more important than config
+	// holds secrets for event driver (as env variable in container)
+	secretMap := map[string]string{}
```

Comment:
`secrets` is already a `map[string]string`, so seems like you don't need that extra map

Reply:
Thanks! Just update it